src/core/core.cpp

```diff
@@ -428,13 +428,18 @@ pid_t wf::compositor_core_impl_t::run(std::string command)
 {
     static constexpr size_t READ_END  = 0;
     static constexpr size_t WRITE_END = 1;
-    pid_t pid;
+
     int pipe_fd[2];
-    pipe2(pipe_fd, O_CLOEXEC);
+    int ret = pipe2(pipe_fd, O_CLOEXEC);
+    if (ret == -1)
+    {
+        LOGE("pipe2: ", strerror(errno));
```

Comment:
I think this error should be a bit more descriptive if we're doing it properly :)

Reply:
See commit